src/lib.rs

```diff
@@ -14,18 +14,29 @@
 //! - `experimental`: Exposes the experimental bignum system library API.
 //!
 //! # Examples
-//! ```ignore
-//! extern crate ewasm_api;
+//! ```
+//! use ewasm_api::prelude::*;
+//!
+//! fn entry() {
+//!     let a: Hash = block_hash(1);
+//!     finish_data(&a.bytes);
+//! }
+//!
+//! ewasm_entry_point!(entry);
+//! ```
 //!
-//! use ewasm_api::{Hash, block_hash, finish_data};
+//! Using lower-level primitives:
+//! ```ignore
+//! use ewasm_api::{types::Hash, block_hash, finish_data};
```

Comment:
It seems after recent changes `types` isn't exposed anymore unless the prelude is used.